thumbnailing/u/framing.go

```diff
@@ -11,15 +11,14 @@ import (
 )
 
 func MakeThumbnail(src image.Image, method string, width int, height int) (image.Image, error) {
-	var result image.Image
-	if method == "scale" {
-		result = imaging.Fit(src, width, height, imaging.Linear)
-	} else if method == "crop" {
-		result = imaging.Fill(src, width, height, imaging.Center, imaging.Linear)
-	} else {
+	switch method {
```

Comment:
Is there a reason for that? (except personal preference)

Reply:
switch statements in Go are weird, so to reduce confusion for folks coming from other languages I try to avoid them. There's also a heavy personal preference (short switch statements are pretty heavy cognitive load)